aws-credentials/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+aws_user: ubuntu
+aws_group: ubuntu
+aws_profile: default
+aws_key_id: ""
+aws_key: ""
+dir_mode: 0700
+file_mode: 0644
```

Comment:
These should probably be prefixed with aws_ as well, since they're specific to this role - possibly something specific like `aws_dir_mode` and `aws_credential_file_mode` or similar